quill-ndbc-postgres/src/test/scala/io/getquill/context/ndbc/postgres/QueryResultTypeNdbcPostgresSpec.scala

```diff
@@ -0,0 +1,107 @@
+package io.getquill.context.ndbc.postgres
+
+import java.util.concurrent.ConcurrentLinkedQueue
+
+import scala.BigDecimal
+import scala.collection.JavaConverters.{ collectionAsScalaIterableConverter, seqAsJavaListConverter }
+import scala.math.BigDecimal.int2bigDecimal
+
+import io.getquill.context.sql.QueryResultTypeSpec
+
+class QueryResultTypeNdbcPostgresSpec extends QueryResultTypeSpec {
+
+  val context = testContext
+  import context._
+
+  val insertedProducts = new ConcurrentLinkedQueue[Product]
```

Comment:
Is there a reason this isn’t just a plain list?

Reply:
The only reason was to keep the consistency with [similar tests](https://github.com/getquill/quill/blob/master/quill-finagle-postgres/src/test/scala/io/getquill/context/finagle/postgres/QueryResultTypeFinaglePostgresSpec.scala#L15). Actually, all the other tests are the same.